### DISCUSSION TITLE
Porsche ambient light more colours

### DISCUSSION THREAD

**Discussion by j0a0pe5r0** [Mr-MIBonk/M.I.B._More-Incredible-Bash#366] [Ideas]
Allow more colors to appear to choose from in the ambient lights menu. I miss the color purple :)

![31b32ce5_78bd_4727_b58a_76733fe1aa6a_41947e1db035dfa8952ba9588b9fb20c085fe178](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/124384648/ba58f9c3-8e2f-4737-9448-a697183688eb)

**Comment by xirixiz** [comment]
+1 😄 

Not sure if this is possible though....
